build_support/install_sionlib.sh

```diff
@@ -2,15 +2,16 @@
 set -euo pipefail
 
 SIONLIBVERSION="1.7.4"
+SIONLIB_INSTALL_PATH="${1:-${HOME}/.cache/csa.install}"
```

Comment:
Are you sure the `csa.install` here is correct? Why not `sionlib.install`?

Reply:
Not shure how you see this, but it has been fixed in https://github.com/nest/nest-simulator/pull/3002/commits/6f937ebcf974557718fd253514e8d38ae25347cc, hence the "(outdated)" above.